### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by anik31**
## Changes made - 
- Do changes as suggested in review
- Add custom scrollbar
- Update hamburger menu

### CODE REVIEW

File: components/button.css [hunk 1]
```diff
@@ -64,7 +65,8 @@
     color: var(--white-color);
     background-color: var(--primary-color);
     border-radius: 50%;
-    padding: 0.8rem 1rem;
+    padding: 0.75rem 0.9rem;
```
Comment (sainath7878) [inline components/button.css:68]: Suggestion: Use px for smaller values.

File: components/navbar.css [hunk 2]
```diff
@@ -81,21 +80,24 @@
...
 }
 
-.hamburger-menu .fa-times{
-    position: absolute;
-    top: 1rem;
-    right: 1rem;
-    font-size: 2rem;
-    color: var(--text-color);
+.hamburger-menu h3{
+    padding: 0.8rem 3rem;
```
Comment (sainath7878) [inline components/navbar.css:94]: Same here

**Review by sainath7878 — APPROVED** [review]
Suggested a small change. Apart from that, every thing looks good!

File: components/button.css (hunk 1 above)
Comment (anik31) [reply components/button.css:68]: Thanks for the suggestion but I prefer using rems

File: components/navbar.css (hunk 2 above)
Comment (anik31) [reply components/navbar.css:94]: okay

**Pull request merged**